generic/tests/jumbo.py

```diff
@@ -69,8 +71,12 @@ def get_ovs_ports(ovs):
     error_context.context("Change all Bridge NICs MTU to %s" %
                           mtu, logging.info)
     for iface in target_ifaces:
-        process.run(host_mtu_cmd % (iface, mtu), shell=True)
-
+        try:
+            process.system_output(host_mtu_cmd % (iface, mtu), shell=True)
+        except process.CmdError as err:
+            if "SIOCSIFMTU" in err.result.stderr:
+                raise exceptions.TestSkipError(
```

Comment:
Please replace this with `test.cancel(...)`.

Reply:
As bug request, she would like to skip test when the ethernet does not support jumbo.

Reply:
@quanwenli test.cancel(...) actually do skip the case as you mean to.